src/style.css

```diff
@@ -0,0 +1,146 @@
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+  background-color: #e5e5e5;
+}
+
+.container {
+  height: 406px;
+  width: 198px;
+    
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
```

Comment:
You can't set width and height for BEM block
Also don't hardcode the height of the main card (check point 7 in checklist)